src/ui/map.js

```diff
@@ -482,11 +498,19 @@ class Map extends Camera {
      * Resizes the map according to the dimensions of its
      * `container` element.
      *
-     * This method must be called after the map's `container` is resized by another script,
+     * Checks if the map container size changed and updates the map if it has changed.
+     * This method must be called after the map's `container` is resized by another script
```

Comment:
Nit: "another script" might not be a little confusing — perhaps change to "programmatically"?